lib/render/editor/editor.js

```diff
@@ -1,9 +1,35 @@
-$(document).ready(() => {
-  $("#custom-editor").html("<gs-element-blockly />");
+$(document).ready(function () {
 
-  const blockly = $("gs-element-blockly")[0];
+  $('#custom-editor').html("<gs-element-blockly />");
 
-  blockly.workspace.addChangeListener(() => {
-    $("#custom-editor-value")[0].value = blockly.generateCode();
-  });
+  function updateCustomEditorValue(blockly) {
+    $('#custom-editor-metadata')[0].value = blockly.workspaceXml;
```

Comment:
We could use `mu-custom-editor-metadata` and `mu-custom-editor-value` (https://github.com/mumuki/mumuki-laboratory/issues/901)

Reply:
Es verdad, no lo hice porque estaba así en labo, pero ahora recuerdo que te comenté lo mismo en el issue que pasaste... y no lo cambiaste :stuck_out_tongue: 

Igual yo lo cambio, don't worry.